components/brave_rewards/browser/publisher_info_database.cc

```diff
@@ -491,6 +492,111 @@ bool PublisherInfoDatabase::InsertOrUpdateActivityInfos(
   return transaction.Commit();
 }
 
+uint64_t PublisherInfoDatabase::GetLastReconcileDate() {
+  DCHECK_CALLED_ON_VALID_SEQUENCE(sequence_checker_);
+
+  bool initialized = Init();
+  DCHECK(initialized);
+  if (!initialized) {
+    return 0;
+  }
+
+  std::string query =
+      "SELECT DISTINCT MAX(date) FROM contribution_info WHERE category = 2";
+  sql::Statement info_sql(db_.GetUniqueStatement(query.c_str()));
+  if (info_sql.Step()) {
+    return info_sql.ColumnInt64(0);
+  }
+  return 0;
+}
+
+uint64_t PublisherInfoDatabase::GetAutoContributeCount(
+    uint64_t reconcile_stamp) {
+  DCHECK_CALLED_ON_VALID_SEQUENCE(sequence_checker_);
+
+  bool initialized = Init();
+  DCHECK(initialized);
+  if (!initialized || reconcile_stamp == 0) {
+    return 0;
+  }
+
+  std::string query =
+    "SELECT DISTINCT COUNT(publisher_id) FROM activity_info AS ai "
+    "WHERE 1 = 1";
+
+  if (reconcile_stamp != 0) {
```

Comment:
I think this is not needed as you have check for `reconcile_stamp == 0` at the top, so we can remove `WHERE 1 = 1` and add it in original query

Reply:
Fixed